README.md

```diff
@@ -397,6 +394,132 @@ const { hash } = await erc165SessionKeyProvider.sendUserOperation({
 });
 ```
 
+### Session Key Validator
+
+```ts
+// 1. Get the default ecdsa validator provider
+const ecdsaProvider = await ECDSAProvider.init({
+  projectId, // zeroDev projectId
+  owner,
+});
+
+// 2. Deploy the Kernel account if not already by sending an empty transaction
```

Comment:
We agreed that this should be removed somehow

Reply:
Yes. On it.

Reply:
Done ✅